src/App.tsx

```diff
@@ -15,48 +17,113 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort();
+  }
```

Comment:
I guess here it is preferable to use localeCompare here (in case our list includes some other starting letters in future)

p.s. And in general switch/case construct is considered to be better if we have some strict sorting options